gradle/libs.versions.toml

```diff
@@ -19,7 +19,7 @@
 
 [versions]
 hadoop = "3.4.0"
-iceberg = "1.6.1"
+iceberg = "1.8.0-SNAPSHOT"
```

Comment:
change to 1.7.1 when released please ref this discussion thread https://github.com/apache/polaris/pull/442#discussion_r1851533111